Lib/test/datetimetester.py

```diff
@@ -6781,6 +6782,47 @@ def test_datetime_from_timestamp(self):
                     self.assertEqual(dt_orig, dt_rt)
 
 
+class ExtensionModuleTests(unittest.TestCase):
```

Comment:
`@cpython_only`?

Reply:
We could, but currently there's mostly no need.  If the extension module ("_datetime") doesn't exist then the "fast" version of the test is never created, and the pure-Python version of the test already always skips.  (See Lib/test/test_datetime.py.)  Also note that `@cpython_only` would mean the test wouldn't be run for Python implementations that do provide a `_datetime` accelerator module.

All that said, I don't see any harm with adding `@cpython_only` on the new test method.  I would certainly only expect it to fail on CPython.  Thanks for bringing this up.